browserify/build.sh

```diff
@@ -0,0 +1,2 @@
+browserify browserify-require-combined.js > nodemods.js
```

Comment:
The build process is:
Run browserify on the browserify-require-combined.js (bad name, I know). This gives us all of the node_modules dependencies in one file. Then we paste that in front of the actual jarr file and we have a pyret standalone with no dependencies.